js/templates/userPage/store.html

```diff
@@ -7,8 +7,8 @@
     </div>
   </div>
   <div>
-    <div class="btnStrip clrT2 clrSh2">
-      <a class="iconBtn clrP clrBr js-toggleListGridView"><i class="ion-android-apps gridViewIcon"></i><i class="ion-navicon listViewIcon"></i></i></a>
+    <div class="clrT2 clrSh2">
```

Comment:
I fixed the missing tooltip here by taking off the btnStrip class. It wasn't needed now that there's only one button, and it had an overflow: hidden on it.

If we want to put tooltips on any other elements that need to keep the btnStrip styling, I can go back and see if it really needs the overflow: hidden, that may be there  to compensate for something we're not doing any more.